static/src/javascripts/projects/atoms/services.js

```diff
@@ -15,11 +15,20 @@ import { viewport } from './services/viewport';
 //    ...
 // }
 
+type FastdomAction = Function => void;
+
+const promisify = (f: FastdomAction) => (g: Function): Promise<any> =>
```

Comment:
can we use fuller variable names? there's no overhead in being 100% explicit here

Reply:
yeah of course

Reply:
ta